src/components/RouteInputForm/RouteInputForm.jsx

```diff
@@ -19,13 +21,23 @@ const RouteInputForm = ({ onStartingPointChange, onDestinationChange }) => {
           ariaLabelledBy="destination-label"
         />
       </div>
+      <div>
+        <label id="arrive-by-label">Arrive By</label>
+        <DateTimeInput
+          value={arriveByValue}
+          onChange={onArriveByDateTimeChanged}
+          ariaLabelledBy="arrive-by-label"
+        />
+      </div>
     </form>
   );
 };
 
 RouteInputForm.propTypes = {
   onStartingPointChange: PropTypes.func.isRequired,
   onDestinationChange: PropTypes.func.isRequired,
+  onArriveByDateTimeChanged: PropTypes.func.isRequired,
+  arriveByValue: PropTypes.string.isRequired,
```

Comment:
good one! with this, we make the arrive-by value controlled even at the route input form level (not only at the datetimeinput level) — at some point it would be good to also convert the starting/destination points into controlled components. not for this pr/ticket tho